doc/programming_model/data_types.md

```diff
@@ -158,93 +164,81 @@ Processor Graphics and Xe Architecture graphics.
  * Xe2-LPG
-   * Intel(R) Graphics for Intel(R) Core(TM) Ultra processors (Series 2) (formerly Lunar Lake)
+   * Intel Graphics for Intel Core Ultra processors (Series 2) (formerly Lunar Lake)
  * Xe2-HPG
-   * Intel(R) Arc(TM) B-Series Graphics (formerly Battlemage)
-
-The following table indicates the data types with performant compute primitives
-for each uArch supported by oneDNN. Unless otherwise noted, all data types have 
-reference support on all architectures.
-
-| uArch   | Supported Data types                                                |
-|:--------|:--------------------------------------------------------------------|
-| Xe-LP   | f32, f16, s8, u8                                                    |
-| Xe-HPG  | f32, f16, bf16, s8, u8                                              |
-| Xe-HPC  | f64, f32, bf16, f16, s8, u8                                         |
-| Xe2-LPG | f64, f32, bf16, f16, s8, u8                                         |
-| Xe2-HPG | f64, f32, bf16, f16, s8, u8                                         |
-| TBA     | f64, f32, bf16, f16, s8, u8, f8\_e5m2, f8\_e4m3, f4\_e2m1, f4\_e3m0 |
-
-
-@note
-  f64 configurations are only supported on GPU engines with HW capability for
-  double-precision floating-point.
-
-@note
-  f8\_e5m2 and f8\_e4m3 compute operations have limited performance through upconversion on
-  Xe-HPC and Xe2 GPUs.
-
-@note
-  f16 operations may be faster with f16 accumulation on GPU architectures older
-  than Xe-HPC. Newer architectures accumulate to f32.
+   * Intel Arc B-Series Graphics (formerly Battlemage)
+ * Xe3-LPG
+   * Intel Arc graphics for future Intel Core Ultra processors (code name Panther Lake)
+
+The following table indicates the data types support for each uArch supported by oneDNN.
+
+| ISA      | f64     | f32     | bf16    | f16     | s8/u8   | f8      | f4_e2m1 | s4/u4   |
+| -------- | ------- | ------- | ------- | ------- | ------- | ------- | ------- | ------- |
+| Xe-LP    |         | `+`     |         | `+`(1)  |         |         |         |         |
+| Xe-HPG   |         | `+`     | `+`     | `+`(1)  |         |         |         | `.`     |
+| Xe-HPC   | `+`     | `+`     | `+`     | `+`     | `+`     | `.`     | `.`     | `.`     |
+| Xe2-LPG  | `+`     | `+`     | `+`     | `+`     | `+`     | `.`     | `.`     | `.`     |
+| Xe2-HPG  | `+`     | `+`     | `+`     | `+`     | `+`     | `.`     | `.`     | `.`     |
+| Xe3-LPG  | `+`     | `+`     | `+`     | `+`     | `+`     | `+`     | `.`     | `.`     |
+
+Legend:
+* `+` indicates full support with compute acceleration.
+* `.` indicates functional support through conversions to higher precision data
+  type or support as a storage format for weights.
+
+Footnotes:
+1. Xe-LP and Xe-HPG architectures do not support f16 operations with f32
+   accumulation. Consider using
+   [relaxed accumulation mode](@ref dev_guide_attributes_accumulation_mode)
+   for the best performance results.
```

Comment:
This comment isn't correct as written. Suggest:
```suggestion
Footnotes:
1. Xe-LP architecture does not natively support f16 operations with f32
   accumulation. Consider using
   [relaxed accumulation mode](@ref dev_guide_attributes_accumulation_mode)
   for the best performance results.
```

Reply:
Looks good.